packages/inference/src/tasks/custom/request.ts

```diff
@@ -13,35 +14,6 @@ export async function request<T>(
 		chatCompletion?: boolean;
 	}
 ): Promise<T> {
-	const { url, info } = await makeRequestOptions(args, options);
-	const response = await (options?.fetch ?? fetch)(url, info);
-
-	if (options?.retry_on_error !== false && response.status === 503) {
-		return request(args, options);
-	}
-
-	if (!response.ok) {
-		const contentType = response.headers.get("Content-Type");
-		if (["application/json", "application/problem+json"].some((ct) => contentType?.startsWith(ct))) {
-			const output = await response.json();
-			if ([400, 422, 404, 500].includes(response.status) && options?.chatCompletion) {
-				throw new Error(
-					`Server ${args.model} does not seem to support chat completion. Error: ${JSON.stringify(output.error)}`
-				);
-			}
-			if (output.error || output.detail) {
-				throw new Error(JSON.stringify(output.error ?? output.detail));
-			} else {
-				throw new Error(output);
-			}
-		}
-		const message = contentType?.startsWith("text/plain;") ? await response.text() : undefined;
-		throw new Error(message ?? "An error occurred while fetching the blob");
-	}
-
-	if (response.headers.get("Content-Type")?.startsWith("application/json")) {
-		return await response.json();
-	}
-
-	return (await response.blob()) as T;
+	const result = await innerRequest<T>(args, options);
```

Comment:
Let's log some warning message as well?

```suggestion
	console.warn("The request method is deprecated and will be removed in a future version of huggingface.js. Use specific task functions instead.")
	const result = await innerRequest<T>(args, options);
```